### PR TITLE
Changed text in client id column in job target grid

### PR DESCRIPTION
**Pull request by ct-anaalbic**
Signed-off-by: ana.albic.comtrade <Ana.Albic@comtrade.com>

Brief description of the PR.
Text in job target grid in client Id column is changed.

**Related Issue**
This PR fixes issue #2269.

**Description of the solution adopted**
Changed "Not Available" text in column client Id in job target grid.

**Screenshots**
/

**Any side note on the changes made**
/

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
I have verified that this PR successfully resolves issue #2269.

File: console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/server/GwtJobTargetServiceImpl.java [hunk 1]
```diff
@@ -68,9 +68,12 @@
 
             // Converto to GWT entity
             for (JobTarget jt : jobTargetList.getItems()) {
-                gwtJobTargetList.add(KapuaGwtJobModelConverter.convertJobTarget(jt));
+                Device device = DEVICE_REGISTRY_SERVICE.find(jt.getScopeId(), jt.getJobTargetId());
+                if (device != null) {
+                    gwtJobTargetList.add(KapuaGwtJobModelConverter.convertJobTarget(jt));
+                    insertClientId(gwtJobTargetList);
```
Comment (muros-ct) [inline console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/server/GwtJobTargetServiceImpl.java]: Why is this moved inside for-loop? It calls insertClientId multiple times with gwtJobTargetList.
